pkg/cloudprovider/providers/azure/azure_vmss.go

```diff
@@ -212,6 +212,8 @@ func (ss *scaleSet) GetInstanceTypeByNodeName(name string) (string, error) {
 }
 
 // GetZoneByNodeName gets cloudprovider.Zone by node name.
+// TODO(feiskyer): Add availability zone support of VirtualMachineScaleSetVM
+// after it is supported in Azure Go SDK.
```

Comment:
would you also add the Azure Go SDK link as it's not ready yet?

Reply:
oops, the pr should be included